primitives/arithmetic/src/per_things.rs

```diff
@@ -69,7 +69,7 @@ macro_rules! implement_per_thing {
 				Self::from_rational_approximation(p, q)
 			}
 
-			/// Converts a fraction into `Permill`.
+			/// Converts from a fraction.
```

Comment:
```suggestion
			/// Converts a fraction into `Self`.
```

Reply:
IMHO "Converts from" sounds weird, but if you say that is correct, you can ignore this as well.